docs/2_registration_project_access_session.md

```diff
@@ -0,0 +1,142 @@
+# 2. Registration and project access
+
+
+
+You should have received a registration link from 
+
+> services-noreply@scilifelab.se
+
+This link is to register a new account, in the testing instance and in a custom unit for this workshop. Open the link and create a new account.
+
+Once that is done. Run
+~~~
+dds auth login
+~~~
+
+Examples on how to use it are available in the [documentation.](https://scilifelabdatacentre.github.io/dds_cli/examples/#authentication-dds-auth)
+
+![enter image description here](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-auth-login.svg)
+
+ - [ ] TASK: Find the email and create an account, login filling your information. Note that the password will not being shown while you are typing it.
+
+# Create a project
+
+~~~
+dds project create --title "<Project Title>" --description "<Project Description>" --principal-investigator "<Email to PI>"
+~~~
+
+> The email specified in the option `--principal-investigator` does not receive any emails; It’s only for information purposes at this time.
+
+[Docs.](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-create)
+When the project is created, you should get an output similar to the one below. Remember the **Project ID**
+
+![enter image description here](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-create.svg)
+
+ - [ ] TASK: Run the create project command, remember to change the files of **title**, **description** and **pi**
```

Comment:
I think those more advanced questions, we can mention them live. Specially if they ask for them.

It will be difficult to account for all the specific options each command has, but all the commands explained have links to the docs, where all the flags are explained. And we can project the documentation while we explain the commands.

Reply:
I think if we're going to talk through it, we should have it written down here (possibly in a box or section labelled Optional). Given that there will be ~15 people, they likely won't remain perfectly in sync with each other and there will be people who won't be able to keep up with the live presentation aspect

Reply:
I will mention it as a note, okay